### ISSUE TITLE
download the [pretrained model] 64_256_upsampler.pt 

### ISSUE THREAD

**Issue by ali-alhwayzee** [openai/guided-diffusion#149]
Hello, is there another mirror for this link to download the `[pretrained model](https://openaipublic.blob.core.windows.net/diffusion/jul-2021/64_256_upsampler.pt)` ?

_WHERE DOWNLOAD ONLY 476 MB AND THEN STOPPED_ 
<img width="268" alt="12122" src="https://github.com/openai/guided-diffusion/assets/50833410/e6715e4f-9992-48aa-8011-6841a52afc8b">
<img width="96" alt="212112" src="https://github.com/openai/guided-diffusion/assets/50833410/bcd99348-d00c-4b7d-9e43-ae2aa2ed7a14">

**Comment by ali-alhwayzee** [comment]
Solved 
After using "!wget -P models/ 
https://openaipublic.blob.core.windows.net/diffusion/jul-2021/64_256_upsampler.pt

“It has been downloaded and is currently working properly

**Comment by Ahmed-wedad** [comment]
is it possible to use this checkpoint as unconditional upsampler ?

**Comment by ali-alhwayzee** [comment]
Yes, the `64_256_upsampler.pt` checkpoint can be used as an unconditional upsampler, provided you handle conditions correctly during inference:
Load the Checkpoint Ensure the model architecture matches the checkpoint and load it into your framework (e.g., PyTorch).

**Comment by Ahmed-wedad** [comment]
thanks .But when i load the checkpoint i should specify that class_cond is True otherwise the checkpoint and the model does not go along . and if i set it True the model expects Class labels.
